mailbox/pom.xml

```diff
@@ -101,7 +101,7 @@
         <geronimo-activation-spec.version>1.1</geronimo-activation-spec.version>
         <slf4j.version>1.7.7</slf4j.version>
         <junit.version>4.11</junit.version>
-        <mockito-core.version>1.9.0</mockito-core.version>
+        <mockito-core.version>2.8.9</mockito-core.version>
```

Comment:
To use Mock rule and mock annotation. Moreover in a general way I think it's better to have newest version, it make more easy finding documentation online.
And why not?